### DISCUSSION TITLE
Install a Old Stable Version via Docker

### DISCUSSION THREAD

**Discussion by PROFLJann** [AzuraCast/AzuraCast#6834] [General]
Hello,

Unfortunately I also had to reinstall Azuracast due to a new server installation,
Unfortunately, I have significant problems with the current version 0.19.3 that I didn't have before.
Is there a way to install an older version via Docker?
I prefer v.0.19.1, which has been very stable in the past.
I would be happy if someone here has an idea.

greeting

**Comment by saintcore** [comment]
After a look at the docker.sh script (which is used to setup azuracast via docker) this should be possible. See https://github.com/AzuraCast/AzuraCast/blob/main/docker.sh --> Line 234 to 238 

![image](https://github.com/AzuraCast/AzuraCast/assets/3920316/295b35be-a624-4fbd-8f0b-50fccec7bdf6)


I assume you should be able to define a var called 'AZURACAST_VERSION' with the exact azuracast-version you want to use within a .env file (same place/folder as the docker.sh script). Azuracast should then use the version defined within this var instead of the current version from rolling-release or stable-release channel. Please note: I've did not test this and didn't find any information about this in the docs. So I assume this is not supported by azuracast and if anything goes wrong it's up to yourself to fix this. I also didn't test this by myself.

**Reply by DJMic** [reply]
I have also problems with latest stable version (0.19.4).
Where to write desired version into docker.sh ? Replace ${AZURACAST_VERSION} at line 254 ?

**Comment by krystim12** [comment]
hello, can someone help me with azuracast?

**Comment by krystim12** [comment]
so I installed azuracast but the problem is only I can listen to the radio, other listeners can't connect, it doesn't work on the public IP, that's the problem, I think, is there anyone who can help me?
